en_us.lang

```diff
@@ -281,83 +281,83 @@ gui.xaero_24h=24h
 gui.xaero_connect_with_auto_msg=Would you like to create the following connection between sub-worlds?
 gui.xaero_disconnect_from_auto_msg=Would you like to delete the following connection between sub-worlds?
-gui.xaero_teleport_not_connected=You are trying to teleport to a sub-world which isn't connected to the current "auto" one. If you are sure that this sub-world is from the same sub-server/world save, then you can enable teleportation by adding a connection in the waypoints menu -> Options -> Add Sub-World Connection. However, if you connect unrelated sub-worlds and teleport, then you might suffocate in a block or die from fall damage, so don't do that.
+gui.xaero_teleport_not_connected=You are trying to teleport to a sub-world that is not connected to the current auto one. If you are sure that this sub-world is from the same world save as the auto one, then you can enable teleportation by adding a connection on the waypoint screen -> Options -> Add Sub-World Connection. But beware, if you connect unrelated sub-worlds and teleport, then there is a good chance you will suffocate in a block or die from fall damage, so be careful and connect only the right ones.
 gui.xaero_ignore_heightmaps=Ignore Server Heightmaps
-gui.xaero_box_ignore_heightmaps=Some servers put custom values in the world heightmaps, which can cause incorrectly rendered maps. Sometimes it can even happen in heavily modded singleplayer worlds. This option should fix such issues at the expense of a bit of performance. Do not use unless you need to. Reenter the world after toggling the option!
+gui.xaero_box_ignore_heightmaps=Some servers put custom values in the world heightmaps, which can result in incorrectly rendered maps. Sometimes it can even happen in heavily modded singleplayer worlds. This option should fix such issues at the expense of a bit of performance. Do not use unless you have to. Reenter the world after toggling this option!
 gui.xaero_error_loading_properties=Error loading server minimap properties. Please retry.
 gui.xaero_light_block=Block
 gui.xaero_light_sky=Sky
 gui.xaero_light_both=All
-gui.xaero_waypoints_bottom=New Waypoints To Bottom
-gui.xaero_box_waypoints_bottom=Put newly created waypoints at the bottom of the waypoints set instead of the default top.
+gui.xaero_waypoints_bottom=New Waypoints to Bottom
+gui.xaero_box_waypoints_bottom=Put newly created waypoints at the bottom of the waypoint list instead of the default top.
 gui.xaero_armour_main_hand=Main Hand Item
-gui.xaero_armour_off_hand=Off-hand Item
+gui.xaero_armour_off_hand=Off-Hand Item
 gui.xaero_minimap_shape=Shape
 gui.xaero_minimap_shape_square=Square
 gui.xaero_minimap_shape_circle=Circle
 gui.xaero_slopes_legacy=Legacy
```

Comment:
Just a quick note here. "Legacy" in computing may also mean obsolete, outdated, stale, etc. If you don't plan to remove the old lighting (IMO the better one ATM) in the near future, what do you think about changing it to "classic" (like in Arrow Color: Classic and Choose a Preset -> Classic)?

Reply:
What do you prefer about the old shading over the new? Would love to know that so I can improve the new one.
I'll consider renaming it to classic. Although it kinda does fit the description of being obsolete/outdated. Hard to say that about the classic arrow color or the classic preset.

Reply:
This is quite complicated. To me, one indisputable advantage of the old lighting is definitely the fact that it has more vibrant colors (look at sand, for example). The slope lighting is not so straightforward, the old block shadows are usually too bright, but the new ones are too dark (however, there is an exception for biomes that have a lot of trees, which look quite nice with the dark shadows). Some biomes definitely look better with the old lighting (especially the snowy ones, desert, savannah), but the majority look OK here and there. Something between the old lighting and the new one would probably be the way to go, I suppose.

For a better understanding what I'm talking about above, here on my "public" OneDrive I have screenshots with the old and the new lighting:
https://1drv.ms/f/s!AnmySc9bJ1N6h3YUhqglKraRgzWC

Reply:
This is quite complicated. To me, one indisputable advantage of the old lighting is definitely the fact that it has more vibrant colors (look at sand, for example). The slope lighting is not so straightforward, the old block shadows are usually too bright, but the new ones are too dark (however, there is an exception for biomes that have a lot of trees, which look quite nice with the dark shadows). Some biomes definitely look better with the old lighting (especially the snowy ones, desert, savannah), but the majority look OK here and there. Something between the old lighting and the new one would probably be the way to go, I suppose.

For a better understanding what I'm talking about above, here on my "public" OneDrive I have screenshots with the old and the new lighting:
https://1drv.ms/f/s!AnmySc9bJ1N6h3ZeOQsRx4LqeLim

Reply:
Ok, thanks! The reason desert looks less "vibrant" is because I added a very slight blue tint (red in the Nether) to the shadows to simulate the reflection of the sky. I might have overdone it. 
The flat surfaces are also shaded (which makes them slightly blue) because the light isn't coming directly from the top. The dark shadows should definitely be this dark in my opinion as it's way more realistic, which is why mountains and trees look better to you. I actually like the blue tint, even in the desert, but I'll look into tweaking the intensity a bit.
Overall, I think you're comparing the 2 too much and very used to the old one. The old one looks bad to me having used the new one for a while. I'll try to bring it back a bit though.

Reply:
Like I said before, the majority of biomes look OK with the old lighting and with the new one too, and in these cases I prefer the old one probably because I simply got used to it. But the sand is definitely too bland (less vibrant colors), and snowy spruce trees look odd (too contrasting shadows), don't they?

Reply:
It's strange to read that because the snow is the most broken part of the old lighting. You could barely see the shadows before.

Reply:
Yes, it is, but I still think that the new shadows are too contrasting on snow. However, I haven't seen any complaints neither on CurseForge nor on Planet Minecraft, so I'll be quiet now.